libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -425,6 +425,23 @@ vector<_T const*> NameAndTypeResolver::cThreeMerge(list<list<_T const*>>& _toMer
 	return result;
 }
 
+string NameAndTypeResolver::similarNameSuggestions(ASTString const& _name) const
```

Comment:
This could be a function in `StringUtils` called something like `quotedAlternativesList`.

Reply:
Done.